### PR TITLE
Correct the order of bad-ssts and --data-dir

### PR DESCRIPTION
**Pull request by chenmin1992**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Exchange the position of bad-ssts nad --data-dir, or it will report `error: Found argument '--data-dir' which wasn't expected, or isn't valid in this context`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/11375
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-control.md [hunk 1]
```diff
@@ -645,7 +645,7 @@ TiKV 中损坏的 SST 文件会导致 TiKV 进程崩溃。在 TiDB v6.1.0 之前
 > 执行此命令前，请保证关闭当前运行的 TiKV 实例。
 
 ```bash
-$ tikv-ctl bad-ssts --data-dir </path/to/tikv> --pd <endpoint>
+$ tikv-ctl --data-dir </path/to/tikv> bad-ssts --pd <endpoint>
```
Comment (Oreoxmt) [inline tikv-control.md:648]:
```suggestion
tikv-ctl --data-dir </path/to/tikv> bad-ssts --pd <endpoint>
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: tikv-control.md (hunk 1 above)
Comment (Connor1996) [reply tikv-control.md:648]: $ is the prompt, should not be deleted

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**